distribution/docs/src/main/resources/content/_configuring/search-options.adoc

```diff
@@ -0,0 +1,23 @@
+:title: Configuring Search Options for ${catalog-ui}
+:type: subConfiguration
+:status: published
+:parent: Configuring ${catalog-ui}
+:order: 00
```

Comment:
Where does this new section end up in the documentation? Would you upload a pdf/html, please?

Reply:
Right. I forgot to ask that...

Reply:
I forgot to check that. We should probably set the `:order:` to `01`